### DISCUSSION TITLE
Another issue updating - 0.38.2

### DISCUSSION THREAD

**Discussion by ZonSoft** [Comfy-Org/ComfyUI-Manager#162] [Q&A]
install: ['https://github.com/FizzleDorf/ComfyUI-AIT']
Download: git clone 'https://github.com/FizzleDorf/ComfyUI-AIT'
[!]
[!] ??? log decoding error ???

One more: all the message boxes regarding data fetching and installation are hidden behind Manager main window.

**Comment by ltdrdata** [comment]
I applied an patch in V0.39. Since it's an issue that I can't reproduce on my end, I can't be sure if it's really effective.

**Reply by ZonSoft** [reply]
It helped. Displays gibberish but works correctly.
[!]  92%|â–ˆâ–ˆâ–ˆâ–ˆâ–ˆâ–ˆâ–ˆâ–ˆâ–ˆâ–�| 22.0/24.0 [00:00<00:00, 50.34it/s]
[!]  96%|â–ˆâ–ˆâ–ˆâ–ˆâ–ˆâ–ˆâ–ˆâ–ˆâ–ˆâ–Œ| 23.0/24.0 [00:00<00:00, 48.27it/s]
[!] 100%|â–ˆâ–ˆâ–ˆâ–ˆâ–ˆâ–ˆâ–ˆâ–ˆâ–ˆâ–ˆ| 24.0/24.0 [00:00<00:00, 50.27it/s]
[!] 100%|â–ˆâ–ˆâ–ˆâ–ˆâ–ˆâ–ˆâ–ˆâ–ˆâ–ˆâ–ˆ| 24.0/24.0 [00:00<00:00, 50.27it/s]
[!] 100%|â–ˆâ–ˆâ–ˆâ–ˆâ–ˆâ–ˆâ–ˆâ–ˆâ–ˆâ–ˆ| 24.0/24.0 [00:01<00:00, 15.82it/s]

**Reply by ZonSoft** [reply]
All message boxes (like "ComfyUI is already up to date with the latest version.") are still hidden below the Manager Setup dialog.

**Reply by ltdrdata** [reply]
Now I understood. What caused this problem. I'll fix it later.

**Comment by ltdrdata** [comment]
This is fixed on V0.39.3
